common/views/components/PageHeader/PageHeader.js

```diff
@@ -188,12 +194,10 @@ const PageHeader = ({
         }}
       >
         {Background}
-        {isFree && (
-          <Layout10>
-            <div className="relative">
-              <FreeSticker />
-            </div>
-          </Layout10>
+        {isFree && isEnhanced && (
+          <Portal id="free-sticker-portal">
+            <FreeSticker />
+          </Portal>
```

Comment:
I'm a little bit confused - why can't we just put the sticker exactly where it needs to be in the dom? I thought portals are primarlyu for when your component has a specific type of DOM it renders but it needs to be outside the containing component? This sticker can live anywhere?

Reply:
Where the sticker is currently, it is in a separate container that is `position: relative`, so it lives in a different stacking context to items in the header – no amount of making-the-number-bigger will get it into the right place. I think awkward z-index is one of the problems that Portals solve:

> React portals are very useful when a parent component has overflow: hidden declared or has properties that affect the stacking context and you need to visually “break out” of its container.

Reply:
But why can't we just move it into the PageLayout, or other component where we don't need to deal with portals or zIndex?

Reply:
We only render it based on an `isFree` prop, so I don't think it can live in the PageLayout. Although regardless of where it lives, it will have to deal with z-index at some point to get around this:

![image](https://user-images.githubusercontent.com/1394592/127137998-f1070ff3-59dd-49ac-a6e7-346b850fbb65.png)

Reply:
Sorry @davidpmccormick I don't think I am being very clear.

My understeanding is that the portal will render this element, just outside of the hierarchy of it's parent component.

What I am trying to understand is - wherever this gets portalled to, why don't we just put it there in the first place?

Reply:
because the component that renders it needs data in the form of a prop that isn't available higher up the DOM

Reply:
…but I think there's still an open question about whether this sticker is a. needed at all or b. needed to be positioned as it is currently creating a z-index headache.